package.json

```diff
@@ -23,10 +23,10 @@
   "license": "Apache-2.0",
   "dependencies": {},
   "devDependencies": {
-    "rewire": "4.0.1",
-    "sinon": "^7.3.2",
-    "standard": "11.0.1",
-    "tape": "^4.9.1",
-    "tmp": "0.0.33"
+    "rewire": "5.0.0",
```

Comment:
Were these upgraded versions determined by an 'npm update' ? Some major versions in there. I will try and get things running here too to triple check all is well.